sorc/orog_mask_tools.fd/lake.fd/enclosure_cnvx.F90

```diff
@@ -32,15 +32,17 @@ PROGRAM testenc
     END PROGRAM
 #endif
 
-
+!> Test whether a given point, p, is inside a convex spherical polygon defined with a series of n vertices
+!! Both the test point and the polygon are specified in latitude and longitude radians
```

Comment:
Punctuation at the end of all sentences please. (Why is this missing from so much of our fortran documentation? Did we used to charge extra for programmers to use the '.' key?)